src/net_processing.cpp

```diff
@@ -5751,7 +5751,7 @@ bool PeerManagerImpl::SendMessages(CNode* pto)
             // Stalling only triggers when the block download window cannot move. During normal steady state,
             // the download window should be much larger than the to-be-downloaded set of blocks, so disconnection
             // should only happen during initial block download.
-            LogPrintf("Peer=%d is stalling block download, disconnecting\n", pto->GetId());
+            LogPrintf("Peer=%d%d is stalling block download, disconnecting\n", pto->GetId(), fLogIPs ? strprintf(" addr=%s", pto->addr.ToStringAddrPort()) : "");
```

Comment:
Should be `Peer=%d%s` (the result of strprintf isn't a number) and `peeraddr=%s` (for better consistency with other `fLogIPs` logs)

Reply:
fixed, thanks.